### PR TITLE
feat: Add discover schema

### PR DESCRIPTION
**Pull request by kejvi-doko**
## Description
Add default event bus schema discoverer. 

## Motivation and Context
This is change is required since the event bus will not initialised after creation.

## Breaking Changes
No breaking changes

## How Has This Been Tested?
- [x] I have executed `pre-commit run -a` on my pull request

### CODE REVIEW

File: variables.tf [hunk 1]
```diff
@@ -78,6 +78,12 @@ variable "bus_name" {
   default     = "default"
 }
 
+variable "discover_description" {
+  description = "Disoverer default schema description"
```
Comment (svenlito) [inline variables.tf:82]:
```suggestion
  description = "Discover default schema description"
```

**Review by svenlito — APPROVED** [review]

File: main.tf [hunk 2]
```diff
@@ -38,6 +38,11 @@ resource "aws_cloudwatch_event_bus" "this" {
   tags = var.tags
 }
 
+resource "aws_schemas_discoverer" "this" {
+  source_arn  = aws_cloudwatch_event_bus.this[0].arn
```
Comment (antonbabenko) [inline main.tf:42]: Add count like on line 35, too

**Review by antonbabenko — CHANGES_REQUESTED** [review]

**Comment by jsecchiero** [comment]
i would like to use this, there are any blockers that are not mentioned in this PR?

**Comment by antonbabenko** [comment]
This issue has been resolved in version 1.16.0 :tada:

**Pull request closed without merging**